plugin/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "kubectl-openebs"
+version = "0.0.7"
```

```suggestion
version = "0.7.0"
```
It's this right? CC @Abhinandan-Purkait

---

Yeah, 0.7.0